src/main/java/inventorysetups/InventorySetupsPlugin.java

```diff
@@ -1694,6 +1684,85 @@ public void toggleFuzzyOnSlot(final InventorySetupsSlot slot)
 		panel.refreshCurrentSetup();
 	}
 
+
+	public void toggleAllFuzzyOnSlot(final InventorySetupsSlot slot)
+	{
+		if (panel.getCurrentSelectedSetup() == null)
+		{
+			return;
+		}
+
+		final InventorySetup owner = slot.getParentSetup();
+		InventorySetupsItem originalItem;
+		//If an additional slot was clicked
+		if (slot.getSlotID() == InventorySetupsSlotID.ADDITIONAL_ITEMS)
+		{
+			Integer keyToToggle = getAdditionalItemKey(slot);
+			if (keyToToggle == null)
+			{
+				return;
+			}
+			else{
+				originalItem = owner.getAdditionalFilteredItems().get(keyToToggle);
+			}
+		}
+		else{
+			originalItem = getContainerFromSlot(slot).get(slot.getIndexInSlot());
+		}
+		if (originalItem.getId() == -1)
+		{
+			return;
+		}
+		boolean fuzz = originalItem.isFuzzy();
```

Comment:
```suggestion
		boolean newFuzzyValue = !originalItem.isFuzzy();
```

And use this value instead of `!fuzz`